diff --git a/index.d.ts b/index.d.ts
index f96e9e6179500578489444201fba6aaa1d65e106..fb67a78992ec854c6927bf73dbc60a98c3ba9abf 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -237,7 +237,7 @@ declare namespace google {
             /**
              * A type representing the action taken by the user to dismiss the dialog.
              */
-            [Response.ACTION]: Action | string;
+            [Response.ACTION]: Action | (string & {});
             /**
              * An array of `DocumentObject`s selected by the user.
              */
