diff --git a/payloads/v10/user.d.ts b/payloads/v10/user.d.ts
index 9c1c5179f2a01a9e1dc9736fd754a00ff98cbdb8..df18494522e951e63b044c0e24d643efc8abcdfa 100644
--- a/payloads/v10/user.d.ts
+++ b/payloads/v10/user.d.ts
@@ -99,6 +99,10 @@ export interface APIUser {
  * @see {@link https://discord.com/developers/docs/resources/user#user-object-user-flags}
  */
 export declare enum UserFlags {
+    /**
+     * @unstable This user flag is currently not documented by Discord and is not supported by discord.js.
+     */
+    None = 0,
     /**
      * Discord Employee
      */
diff --git a/payloads/v10/user.js b/payloads/v10/user.js
index 0ed4fc9afc1376c6e815be98c3878559cde77f56..ec7ec4eb08c07826e2aa656d2b38a0106e312680 100644
--- a/payloads/v10/user.js
+++ b/payloads/v10/user.js
@@ -9,6 +9,10 @@ exports.ConnectionVisibility = exports.ConnectionService = exports.UserPremiumTy
  */
 var UserFlags;
 (function (UserFlags) {
+    /**
+     * @unstable This user flag is currently not documented by Discord and is not supported by discord.js.
+     */
+    UserFlags[UserFlags["None"] = 0] = "None";
     /**
      * Discord Employee
      */
