diff --git a/build/src/crypto/node/crypto.js b/build/src/crypto/node/crypto.js
index 38e36331a765e3fa9c6cd144e01644b3710e9367..7b9fdf3e5a9647f87f7133153a215dc3529ca944 100644
--- a/build/src/crypto/node/crypto.js
+++ b/build/src/crypto/node/crypto.js
@@ -14,7 +14,7 @@
 // limitations under the License.
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.NodeCrypto = void 0;
-const crypto = require("crypto");
+const crypto = require("crypto-browserify");
 class NodeCrypto {
     async sha256DigestBase64(str) {
         return crypto.createHash('sha256').update(str).digest('base64');
diff --git a/package.json b/package.json
index 39bc323480871b38fb0e155e6c7cd05fb52f2ced..6552c2226778bcc213ed6b85276946a15eb98148 100644
--- a/package.json
+++ b/package.json
@@ -23,7 +23,8 @@
     "gcp-metadata": "^7.0.0",
     "google-logging-utils": "^1.0.0",
     "gtoken": "^8.0.0",
-    "jws": "^4.0.0"
+    "jws": "^4.0.0",
+    "crypto-browserify": "^3.12.1"
   },
   "devDependencies": {
     "@types/base64-js": "^1.2.5",
