diff --git a/source/split.d.ts b/source/split.d.ts
index 2b935c90a2d3735298edf50c22442aabadc05fe1..b3095ac02f44f44e9b9df28e9bb1668fd993d5c6 100644
--- a/source/split.d.ts
+++ b/source/split.d.ts
@@ -22,7 +22,9 @@ array = split(items, ',');
 export type Split<
 	S extends string,
 	Delimiter extends string,
-> = SplitHelper<S, Delimiter>;
+> = Delimiter extends infer _Delimiter extends string
+	? SplitHelper<S, _Delimiter>
+	: never;
 
 type SplitHelper<
 	S extends string,
